src/pages/config/config-details.js

```diff
@@ -91,16 +91,13 @@ export function ConfigDetailsPage() {
           ? `Config Details for ${configDetails.name}`
           : "Config Details"
       }
+      button={<Link className="underline mr-3"
+        to={`/topology`}>
```

Comment:
@gyanpgupta why the change from /config to /topology ?

Reply:
@moshloop , before it was redirecting to previous page but as it is mentioned in the task to remove back button & add a link which redirect to home page and '/topology' is the route for the homepage